pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/UsingPreparedStatementAndMetaDataObjects/DAOClass.java

```diff
@@ -0,0 +1,123 @@
+package com.subho.wipro.pjp.tm08.UsingPreparedStatementAndMetaDataObjects;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+
+public class DAOClass {
+	
+	private Connection conn;
+	PreparedStatement pstmt = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)